frontend/views/pages/DesignSystem.vue

```diff
@@ -19,22 +19,22 @@ page(
       i18n(tag='h2' class='card-header') Typography
 
       h3 Titles
-
+      p For ease of use, all heading tags come pre-styled, but remember: #[strong Heading Scales ≠ H# Tags]
+      | Apply a H tag that makes sense in that context, not the one that matches the design.
+      | In other words, it's okay to code #[code p.is-4] or #[code h3.is-2] when semanticaly the markup doesn't match the pre-styled.
+      br
+      br
       table
         thead
           th code
           th demo
         tr(v-for='(scale, index) in [1, 2, 3, 4]' :key='index')
           td
-            pre h{{scale}} or .title.is-{{scale}}
-          td.title(:class='`is-${scale}`') This is an H{{scale}} title
+            pre h{{scale}} or .is-{{scale}}
+          td(:class='`is-${scale}`') This is an H{{scale}} title
```

Comment:
@taoeffect and @pieer 
In our DS, the util classes usually follow this convention `has-[property]-[variant]`. ex (`has-text-bold` or `has-background-success` or `has-text-1`. The titles are the only one `.is-[n]`. This comes like this directly from Bulma, but I believe it should be more explicit. `has-title-[n]`. What do you think?

Reply:
For the record I repeat here: If you prefer to use 'is-title-1' instead of 'is-1', go for it!